src/main/java/uk/gov/hmcts/darts/arm/client/ArmClientConfig.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.darts.arm.client;
+
+import feign.RequestTemplate;
+import feign.codec.EncodeException;
+import feign.codec.Encoder;
+import groovy.util.logging.Slf4j;
+import org.springframework.context.annotation.Bean;
+import uk.gov.hmcts.darts.arm.client.model.ArmTokenRequest;
+
+@Slf4j
+public class ArmClientConfig {
```

Comment:
Is this class missing the spring `@Configuration`, since it uses the `@Bean` annotations below?
Also wondering, should it be under a `config` sub package as we normally like doing that

Reply:
Not in this case - adding `@Configuration` would make this Encoder configration apply to all Feign clients, whereas here we only wish to override the Encoder behaviour for the Arm Token client only.

Reply:
Happy to move into a `config` dir.